examples/nodes/tasks/combine.tick

```diff
@@ -0,0 +1,10 @@
+dbrp "telegraf"."autogen"
+
+stream
+    |from()
+        .measurement('system')
+        .groupBy(*)
+    |combine(lambda: isPresent("load1"), lambda: isPresent("uptime"))
```

Comment:
maybe this one could compare the cpu-total to all other cpus?

```
 |combine(lambda: "cpu' == 'cpu-total', lambda: TRUE)
     .as('total', 'cpu')
```

Reply:
fixed